### DISCUSSION TITLE
Load Power and Inverter Power confused?

### DISCUSSION THREAD

**Discussion by ICEAndreas** [nathanmarlor/foxess_modbus#649] [Support]
It seams like that Load Power and Inverter Power are confused
![Screenshot_20240717_210147_Home Assistant](https://github.com/user-attachments/assets/48870f1f-bed7-445b-a8dc-5308891262ee)
.

**Comment by FozzieUK** [comment]
In what way are they confused?, load power (r/s/t) reports the calculated load of each of the phases, whereas inverter power (r/s/t) is the amount of power the inverter is outputting - this could be going to export, battery or the load and will include any losses of the inverter/bms/batteries.

**Comment by ICEAndreas** [comment]
Load Power is the power, that goes into the house?! The "Load Energy today" with 9,3kWh seems ok, but not the power. It's too less. I have a base load of 300-500W in the house and not 44W. The Inverter Power is the power, that the inverter use itself I think?! So the Load Power must be the Inverter Power und the Inverter Power must be the Load Power.

**Reply by canton7** [reply]
No, Inverter Power is not the power used by the inverter. It is the power output by the inverter.

**Reply by ICEAndreas** [reply]
Okay, and what is the Load Power?

**Reply by canton7** [reply]
The power drawn by your house, regardless of whether that power was supplied from your inverter or the grid. See https://github.com/TonyM1958/HA-FoxESS-Modbus/wiki/How-does-it-work%3F

**Reply by FozzieUK** [reply]
The complication here is that load power is not measured, it's calculated in near real time by the inverter from the sum of all the inputs and outputs and in particular at low loads it's not a perfect sum - particularly on 3 phase systems.

This is made all the more complex by the fact that on different inverters and differing firmware versions, Fox appear to change how and where they incorporate the losses of DC/AC (discharge/solar) and AC/DC (charge) conversion and inverter/bms/battery losses.

There's a bit of discussion in other threads about how you measure load accurately on a 3 phase, some have resorted to fitting an energy monitor to measure it more precisely - it would seem that on the latest H3 firmware Foxess have added an installer option to set a grid compensation value (as some other manufacturers do) to 'fix' the load differences.

At high loads, load power will be _virtually_ spot on, but at low loads it can be a few hundred watts out - you can see the discussion here on grid compensation https://github.com/nathanmarlor/foxess_modbus/discussions/645#discussioncomment-10082935

**Reply by horstvogel** [reply]
load Energy Total is household consumption? Are the values ​​correct now? it is a value that is mesasured? 

But load power is not “right” at the moment, but with an update it will soon be “better”.

thanks.
